src/olympia/addons/management/commands/process_addons.py

```diff
@@ -42,6 +43,9 @@
             Q(**{current_autoapprovalsummary + 'verdict': amo.AUTO_APPROVED}) &
             ~Q(**{current_autoapprovalsummary + 'confirmed': True})
         ]},
+    'constantly_recalculate_post_review_weight': {
+        'method': recalculate_post_review_weight,
```

Comment:
Maybe worth noting somewhere that this will recalculate the entire weight, not just replace the bits that may have changed like abuse report count and negative ratings count. It could be costly.

Reply:
Indeed, I thought it's worth optimizing once we see that it takes too long but keep it simple at first.

Reply:
Added a comment and will monitor this once it's live.